derive_builder_core/src/macro_options/darling_opts.rs

```diff
@@ -810,7 +814,14 @@ impl<'a> FieldWithDefaults<'a> {
                 // Disabled fields become a PhantomData in the builder.  We make that field
                 // non-public, even if the rest of the builder is public, since this field is just
                 // there to make sure the struct's generics are properly handled.
-                || (!self.field_enabled()).then(|| Cow::Owned(syn::Visibility::Inherited)),
+                || {
+                    // We would prefer to use bool::then but that's only available in Rust 1.50, not in our MSRV
```

Comment:
```suggestion
```

This is true, but the savings of switching to `then` are small enough that it's not worth the added line for the comment.